chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/pendingListing/reviewer/TestStandardAndOptionalStandardReviewer.java

```diff
@@ -0,0 +1,34 @@
+package gov.healthit.chpl.validation.pendingListing.reviewer;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import gov.healthit.chpl.dto.listing.pending.PendingCertificationResultDTO;
+import gov.healthit.chpl.dto.listing.pending.PendingCertifiedProductDTO;
+import gov.healthit.chpl.util.ErrorMessageUtil;
+import gov.healthit.chpl.util.Util;
+
+@Component("pendingTestStandardAndOptionalStandardReviewer")
+public class TestStandardAndOptionalStandardReviewer implements Reviewer {
+    private ErrorMessageUtil msgUtil;
+
+    @Autowired
+    public TestStandardAndOptionalStandardReviewer(ErrorMessageUtil msgUtil) {
+        this.msgUtil = msgUtil;
+    }
+
+    @Override
+    public void review(PendingCertifiedProductDTO listing) {
+        listing.getCertificationCriterion().stream()
+            .filter(cert -> (cert.getMeetsCriteria() != null && cert.getMeetsCriteria().equals(Boolean.TRUE)))
+            .filter(cert -> (cert.getTestStandards() != null && cert.getTestStandards().size() > 0))
+            .filter(cert -> (cert.getOptionalStandards() != null && cert.getOptionalStandards().size() > 0))
```

Comment:
This is fine and obviously works but I feel like it would be more clear to have a simple method called something like "hasBothTestStandardsAndOptionalStandards()" that does what the last two filters do here. Just to make it very readable about what problem you are checking for. Up to you if you agree, I wouldn't hold up the PR for that.